### ISSUE TITLE
Infinite patch loop

### ISSUE THREAD

**Issue by Crazymine-hub** [PhantomGamers/SteamFriendsPatcher#14]
The friends window keeps opening and closing with the default and patched style alternating.
It also keeps getting patched each time.

Exact process:
 * friends appears in default style (online, listing friends)
 * Message: "Successfully patched friends"
 * friends disappears
 * friends reappears in patched style (connecting to server animation)
 * friends disappears
repeat

**Comment by SgtBreadStick** [comment]
Also having this issue with friends. The patcher just doesn't work anymore even with "Steam Beta" ticked in the options.

**Comment by tehuno** [comment]
You must tick Patch Library [BETA] and Steam Beta in order for it to work

**Comment by SgtBreadStick** [comment]
Did that. While it patches the library it does not patch the Friends/Chat properly, it just does what Crazymine-hub has said. There's no workaround for the issue at the moment by the looks of it.

**Comment by mutageneral** [comment]
Same problem here

**Comment by tehuno** [comment]
It literally just worked for me with "patch library" and "steam beta" both checked in settings.

**Comment by PhantomGamers** [comment]
> It literally just worked for me with "patch library" and "steam beta" both checked in settings.

neither of these settings affect friends list patching

**Comment by LilDevil71** [comment]
Hello, I exactly got the same problem as this topic, is there any fix yet? Or a way to make it working somehow? Thanks.

**Comment by TacoRaptor67** [comment]
Mine wasn't working so I came here to comment on this, but after checking Patch Library it seems to have fixed it. Not at all saying your issue isn't happening of course.

**Comment by PhantomGamers** [comment]
> Mine wasn't working so I came here to comment on this, but after checking Patch Library it seems to have fixed it. Not at all saying your issue isn't happening of course.

This is a fluke, library patching has nothing to do with friends list patching.

**Comment by TacoRaptor67** [comment]
> This is a fluke, library patching has nothing to do with friends list patching.
>
Yeah.. started again after a restart. I just loaded up Windows, and it was infinitely patching. I closed out of both the Patcher as well as Steam, started up the Patcher, started up Steam, and it properly patched it this time. So I dunno what the issue could be.

**Comment by tehuno** [comment]
> > It literally just worked for me with "patch library" and "steam beta" both checked in settings.
> 
> neither of these settings affect friends list patching

On the main page of the github it says that steam beta needs to be checked in recommended settings so idk what you're saying.

**Comment by PhantomGamers** [comment]
> On the main page of the github it says that steam beta needs to be checked in recommended settings so idk what you're saying.

Yes, for library patching. Nothing to do with the friends list.

**Comment by Sartharon** [comment]
I just let it run through at some point it's able to patch friends. And if you look closely it's patching different files every time it restarts Friends
